### ISSUE TITLE
Retina/Scaled resolution rendering issue

### ISSUE THREAD

**Issue by evelyndooley** [m64p/mupen64plus-GLideN64#119]
I have a Mac running OSX 10.14.2. My display has a native resolution of 2560x1600 and I use OSX's built in "retina" scaling feature with an effective resolution of 1680x1050. If I launch a game while in this mode, the actual game only fills up a quarter of the window. 

<img width="752" alt="screen shot 2019-01-03 at 3 32 30 pm" src="https://user-images.githubusercontent.com/31457788/50660327-5e552100-0f6d-11e9-8b8d-5b8c15931a0a.png">

When I change it to a non-scaled resolution it works as intended. I also did not have this problem with m64py, but I had other issues with that one. You may be able to replicate it using Windows' DPI scaling feature, although that one works a little differently. I'll look into making a PR for this but if anyone else wants to work on it I can test it.

**Comment by loganmc10** [comment]
Yeah this is something that would require a PR, I don't have a Mac
